symfony/client/src/orangehrmAttendancePlugin/pages/view-my-attendance.scss

```diff
@@ -14,16 +14,16 @@
 // if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
 // Boston, MA  02110-1301, USA
 
-.orangehrm-my-attendance {
-  ::v-deep(.card-header-slot) {
-    border-bottom: $oxd-input-control-border--active;
-    margin-bottom: 5px;
-  }
-  ::v-deep(.card-header-slot-content .oxd-table-cell-actions) {
-    justify-content: flex-end;
-  }
-}
+// .orangehrm-my-attendance {
+//   ::v-deep(.card-header-slot) {
+//     border-bottom: $oxd-input-control-border--active;
+//     margin-bottom: 5px;
+//   }
+//   ::v-deep(.card-header-slot-content .oxd-table-cell-actions) {
+//     justify-content: flex-end;
+//   }
+// }
 
-.orangehrm-header-total {
-  margin-left: auto;
-}
+// .orangehrm-header-total {
+//   margin-left: auto;
+// }
```

Comment:
why commented code?

Reply:
it's not needed now but chamara add these styles I'm not sure the future may be styles needed or not?